doc/DEVELOPERS.rst

```diff
@@ -50,25 +50,11 @@ activate a venv::
 
     git clone https://github.com/rst2pdf/rst2pdf
     cd rst2pdf
-    python3 -m venv .venv
-    . .venv/bin/activate
+    uv sync --all-extras
 
-Ensure that setuptools and pip are up to date::
+If you don't have ``uv``, please see `the installation docs <https://docs.astral.sh/uv/getting-started/installation/>`_
 
-    pip install --upgrade setuptools pip
-
-Now you can install rst2pdf from this source code::
-
-    pip install -c requirements.txt -e .[aafiguresupport,mathsupport,plantumlsupport,rawhtmlsupport,sphinx,svgsupport,tests]
-
-Note, that on Apple Silicon Macs, you may need this to get tests passing::
-
-    pip install reportlab==3.6.12 --force-reinstall --no-cache-dir --global-option=build_ext
```

Comment:
Note this special case should no longer be required after `reportlab>=4.0.0` which is pure Python according to their [release notes](https://docs.reportlab.com/install/open_source_installation/):

> Since ReportLab 4.0 we have moved to pure Python!